frontend/javascripts/oxalis/view/action_bar_view.js

```diff
@@ -181,6 +181,7 @@ class ActionBarView extends React.PureComponent<Props, State> {
           )}
           {showVersionRestore ? VersionRestoreWarning : null}
           <DatasetPositionView />
+          <Icon type="warning" />
```

Comment:
What is this warning icon about?

Reply:
Whoops, that creeped in when I experimented with different warning designs for missing magnifications ^^